src/app/app.py

```diff
@@ -61,10 +82,16 @@ async def get_schema(survey_id: str, version: str) -> dict:
     logger.info("Getting schema metadata...")
     logger.debug(f"Input data: survey_id={survey_id}, version={version}")
 
+    try:
```

Comment:
I wouldn't have this validation logic in app.py, ideally we'd abstract validation logic into a separate Validator class

Reply:
Moved logic to a separate file and class